### ISSUE TITLE
method which name is underscore cannot use.

### ISSUE THREAD

**Issue by soranoba** [nicklockwood/SwiftFormat#977]
Run swiftformat with no config makes the following changes, but these changes fail to compile.

```diff
@@ -58,7 +58,7 @@ open class Store<State> {
     /// Returns the BindingTarget for the keyPath.
     /// - Parameter keyPath: KeyPath to bind.
     /// - Returns: The BindingTarget.
-    open func `_`<Value>(_ keyPath: KeyPath<State, Value>) -> BindingTarget<State, Value> {
+    open func _<Value>(_ keyPath: KeyPath<State, Value>) -> BindingTarget<State, Value> {
         return BindingTarget(store: self, keyPath: keyPath)
     }
 ```

**Comment by nicklockwood** [comment]
@soranoba thanks for reporting this. It's caused by the `redundantBackticks` rule, so you can just disable that with `--disable redundantBackticks` as a workaround for now.

**Comment by soranoba** [comment]
I see. Thanks many.

**Comment by nicklockwood** [comment]
@soranoba fixed in 0.48.10
